tests/app/celery/v3/test_notification_tasks.py

```diff
@@ -39,15 +38,16 @@ def test_v3_process_notification_no_template(notify_db_session, mocker, sample_s
     v3_process_notification(request_data, sample_service.id, None, KEY_TYPE_TEST)
     v3_send_email_notification_mock.assert_not_called()
 
-    query = select(Notification).where(Notification.id == request_data["id"])
-    notification = notify_db_session.session.execute(query).one().Notification
-    assert notification.status == NOTIFICATION_PERMANENT_FAILURE
-    assert notification.status_reason == "The template does not exist."
+    query = select(NotificationFailures).where(NotificationFailures.notification_id == request_data["id"])
+    notification_failure = notify_db_session.session.execute(query).one()[0]
+    body = notification_failure.body
+
+    assert body.get('status') == NOTIFICATION_PERMANENT_FAILURE
+    assert body.get('status_reason') == "The template does not exist."
```

Comment:
please change to double-quotes to be consistent

Reply:
done